lua/telescope/command.lua

```diff
@@ -0,0 +1,90 @@
+local themes = require('telescope.themes')
+local builtin = require('telescope.builtin')
+local extensions = require('telescope._extensions').manager
+local config = require('telescope.config')
+local command = {}
+
+-- convert command line string arguments to
+-- lua number boolean type and nil value
+local function convert_user_opts(user_opts)
+  local default_opts = config.values
+
+  local _switch = {
+    ['boolean'] = function(key,val)
+      if val == 'false' then
+        user_opts[key] = false
+      end
+      user_opts[key] = true
+    end,
+    ['number'] = function(key,val)
+      user_opts[key] = tonumber(val)
+    end,
+    ['string'] = function(key,val)
+      if val == 'nil' then
+        user_opts[key] = nil
+      end
+    end
+  }
```

Comment:
Lets pull that table out of that scope. No need to define it more then once.

Reply:
hmm create a new file just for this `_switch`

Reply:
No not a new file. Just out of that function scope.

Reply:
hmm value in `_switch` is a closure function use variable `user_opts`. remove out function. It will receive three arguments..

Reply:
Oh yeah. I am dumb i haven't seen that 😆

Reply:
I will test your PR later.